test/Microsoft.ML.TestFramework/RemoteExecutorTestBase.cs

```diff
@@ -0,0 +1,243 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.ML.TestFramework
+{
+    /// <summary>Base class used for all tests that need to spawn a remote process.</summary>
+    public abstract partial class RemoteExecutorTestBase
+    {
+        /// <summary>The name of the test console app.</summary>
+        protected static readonly string TestConsoleApp = Path.GetFullPath(@"RemoteExecutorConsoleApp.dll");
+#if NET461
```

Comment:
I commented on your other PR, but with [https://github.com/dotnet/sdk/pull/2073](https://github.com/dotnet/sdk/pull/2073), this can be changed to `#if NETFRAMEWORK`